src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -246,7 +219,7 @@ export default function TokenTable({
               <ClickableText onClick={() => handleSort(SORT_FIELD.NAME)}>
                 <Trans>Name {arrow(SORT_FIELD.NAME)}</Trans>
               </ClickableText>
-              <ClickableText disabled={!account} onClick={() => (account ? handleSort(SORT_FIELD.BALANCE) : false)}>
+              <ClickableText disabled={true} /* onClick={() => (account ? handleSort(SORT_FIELD.BALANCE) : false)} */>
```

Comment:
Any reason to keep the comment?
Otherwise:

```suggestion
              <ClickableText disabled>
```

Reply:
I've commented it out to disable the sorting on Balance for now, but we will probably use this in the future.